.travis.yml

```diff
@@ -32,7 +32,7 @@ env:
         - ASTROPY_USE_SYSTEM_PYTEST=1
 
         - CONDA_DEPENDENCIES='Cython click scipy healpy matplotlib pyyaml pandas naima sherpa libgfortran regions reproject pandoc ipython iminuit'
-        - CONDA_DEPENDENCIES_OSX='Cython click scipy healpy matplotlib pyyaml pandas naima sherpa regions reproject pandoc ipython iminuit'
+        - CONDA_DEPENDENCIES_OSX='Cython click numpy scipy healpy matplotlib pyyaml pandas naima sherpa regions reproject pandoc ipython iminuit'
```

Comment:
I am not sure if this is necessary. Since you are using ci-helpers, you can control it with `NUMPY_VERSION`. See https://github.com/astropy/astropy/blob/master/.travis.yml